config/ubuntu/waagent.conf

```diff
@@ -82,6 +82,9 @@ OS.SshDir=/etc/ssh
 # Enable RDMA management and set up, should only be used in HPC images
 # OS.EnableRDMA=y
 
+# Enable RDMA kernel update, should only be used with Ubuntu
```

Comment:
I'll change the wording. As of today, this option is only effective under Ubuntu.